tests/unit/test_charm.py

```diff
@@ -68,7 +69,7 @@ def test_charm_reaches_active_status(urlopen_mock):
         )
         harness.begin()
         harness.charm.on.config_changed.emit()
-        harness.model.unit.status == ops.ActiveStatus()
+        assert harness.model.unit.status == ops.ActiveStatus()
```

Comment:
should we also do this check by name?

Reply:
Not really, this one doesn't contain a status message